smoothie-sample/src/main/java/com/example/smoothie/SimpleApp.java

```diff
@@ -22,6 +22,10 @@ public void onCreate() {
     MemberInjectorRegistryLocator.setRootRegistry(new com.example.smoothie.MemberInjectorRegistry());
     FactoryRegistryLocator.setRootRegistry(new com.example.smoothie.FactoryRegistry());
 
+    initToothpick();
+  }
+
+  public void initToothpick() {
     Scope appScope = Toothpick.openScope(this);
     appScope.installModules(new SmoothieApplicationModule(this));
```

Comment:
I am wondering if it would not be more clear to only install modules here and pass the scope as a param. 
What needs to be called from the test is only this, not the creation of the scope itself.

But that's really just a detail.